commons/ava_testsuite/five_node_staking_gecko_network_tests.go

```diff
@@ -118,10 +189,56 @@ func (test FiveNodeStakingNetworkGetValidatorsTest) Run(network interface{}, con
 }
 
 func (test FiveNodeStakingNetworkGetValidatorsTest) GetNetworkLoader() (testsuite.TestNetworkLoader, error) {
-	return fixed_gecko_network.NewFixedGeckoNetworkLoader(5, 1, true)
+	return fixed_gecko_network.NewFixedGeckoNetworkLoader(5, 5, true)
 }
 
 func (test FiveNodeStakingNetworkGetValidatorsTest) GetTimeout() time.Duration {
 	return 30 * time.Second
 }
 
+
+func createAndSeedXChainAccountFromGenesis(client *gecko_client.GeckoClient, username string, password string, amount int) (string, error) {
```

Comment:
This looks like a copy-paste of the same method inside RpcManager. Is that intentional??